api/rest/v1/bulk_observations_point_linked.md

```diff
@@ -0,0 +1,298 @@
+nav_order: 9
+parent: REST (v1)
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/point/linked
+---
+
+# /v1/bulk/observations/point/linked
+
+Retrieve single [observations](/glossary.html#observation) of multiple [variables](/glossary.html#variable) at a set date for entities linked to a parent entity by the same property.
+
+This is useful for retrieving observations for all places within a parent place. For example, this could be getting the population of women in 2018 for all states in the United States.
+
+<div markdown="span" class="alert alert-info" role="alert" style="color:black; font-size: 0.8em">
+   <span class="material-icons md-16">info </span><b>Note:</b><br />
+   Currently, this endpoint only supports the `containedInPlace` property and `Place` entities. Support for other properties and entity types will be added in the future.
+</div>
+
+<div markdown="span" class="alert alert-warning" role="alert" style="color:black; font-size: 0.8em">
+    <span class="material-icons md-16">info </span><b>See Also:</b><br />
+    To get single observations for a single place, see [/v1/observations/point](/api/rest/v1/observations/point).<br />
+    To get a series of observations for all places within a parent place, see [/v1/bulk/observations/series/linked](/api/rest/v1/observations/series/linked).
+</div>
+
+## Request
+
+<div class="api-tab">
+  <button id="get-button" class="api-tablink" onclick="openTab(event, 'GET-request')">GET Request</button>
+  <button id="post-button" class="api-tablink" onclick="openTab(event, 'POST-request')">POST Request</button>
+</div>
+
+<div id="GET-request" class="api-tabcontent api-signature">
+https://api.datacommons.org/v1/bulk/observations/point/linked?linked_property=containedInPlace&linked_entity={parent_place_dcid}&entity_type={place_type}&variables={variable_dcid_1}&variables={variable_dcid_2}&date={date}&key={your_api_key}
+</div>
+
+<div id="POST-request" class="api-tabcontent api-signature">
+URL:
+https://api.datacommons.org/v1/bulk/observations/point/linked
+
+Header:
+X-API-Key: {your_api_key}
+
+JSON Data:
+{
+  "linked_property": "containedInPlace",
+  "linked_entity": "{parent_place_dcid}"
+  "entity_type": "{place_type}",
+  "date": "{date}",
+  "variables": [
+    "{variable_dcid_1}",
+    "{variable_dcid_2}",
+    ...
+  ]
+}
+</div>
+
+<script src="/assets/js/syntax_highlighting.js"></script>
+<script src="/assets/js/api-doc-tabs.js"></script>
+
+### Path Parameters
+
+There are no path parameters for this endpoint.
+
+### Query Parameters
+
+| Name                                               | Type | Description               |
+| -------------------------------------------------- | ---- | ------------------------- |
+| key <br /> <required-tag>Required</required-tag>   | string | Your API key. See the [page on authentication](/api/rest/v1/getting_started#authentication) for a demo key, as well as instructions on how to get your own key. |
```

Comment:
I found table cell line wrapping can be done like:

| cell1          | cell 22222222       |
: cell2         : is long long            :

Worth checking if can find a linter do this

Reply:
This doesn't seem to be supported in kramdown. using `:` breaks the table when testing locally, but I'll continue to look into this!